Marlin/mesh_bed_leveling.h

```diff
@@ -41,6 +41,11 @@
     static FORCE_INLINE float get_probe_y(int8_t i) { return MESH_MIN_Y + (MESH_Y_DIST) * i; }
     void set_z(const int8_t px, const int8_t py, const float z) { z_values[py][px] = z; }
 
+    bool has_mesh() { return TEST(status, 0); }
```

Comment:
Applying the constants here…
```cpp
FORCE_INLINE bool has_mesh() { return TEST(status, MBL_HAS_MESH); }
void has_mesh(bool onOff) { onOff ? SBI(status, MBL_HAS_MESH) : CBI(status, MBL_HAS_MESH); }
FORCE_INLINE bool is_active() { return TEST(status, MBL_ACTIVE); }
void is_active(bool onOff) { onOff ? SBI(status, MBL_ACTIVE) : CBI(status, MBL_ACTIVE); }
```
In some paradigms it's considered better practice to name setters with "`set_`" on the front to more readily differentiate them from the getters. Thus, `has_mesh` to get, and `set_has_mesh` to set.